extensions/amp-recaptcha-input/0.1/amp-recaptcha-service.js

```diff
@@ -255,9 +252,14 @@ export class AmpRecaptchaService {
    */
   getRecaptchaFrameSrc_() {
     if (getMode().localDev || getMode().test) {
-      return tryResolve(() => {
-        return getDevelopmentBootstrapBaseUrl(this.win_, 'recaptcha');
-      });
+      return ampToolboxCacheUrl.createCurlsSubdomain(this.win_.location.href)
+          .then(curlsSubdomain => {
+            return 'http://' + curlsSubdomain +
+          '.recaptcha.localhost:8000/dist.3p/' +
+          (getMode().minified ? '$internalRuntimeVersion$/recaptcha'
+            : 'current/recaptcha.max') +
```

Comment:
@zhouyx do you think this is still needed? I think we've got rid of `.max.` pattern.

Reply:
Unfortunately we can't remove `.max.` pattern today. We haven't added proxy to frame.html or nameframe.html`. We should be able to remove them after adding proxy to them later on. 

@torch2424 Feel free to leave the .max as it is. Thanks